src/manage_sql_nvts.c

```diff
@@ -1238,8 +1236,15 @@ nvti_from_vt (entity_t vt)
   detection = entity_child (vt, "detection");
   if (detection)
     {
+      const gchar *qod;
+
       nvti_set_detection (nvti, entity_text (detection));
-      nvti_set_qod_type (nvti, entity_attribute (detection, "qod_type"));
+
+      qod = entity_attribute (detection, "qod");
+      if (qod == NULL)
+        nvti_set_qod_type (nvti, entity_attribute (detection, "qod_type"));
```

Comment:
You don't need to set qod_type when qod is given?

Reply:
It is one or the other. There should not be both at the same time. OSPD also considers one of them, not both.